site/docs/v1/tech/apis/tenants.adoc

```diff
@@ -20,6 +20,7 @@ possibility of collision.
 The following APIs are provided to manage Tenants.
 
 * <<Create a Tenant>>
+* <<Create a New Tenant From an Existing Tenant>>
```

Comment:
Same as the application comment, use sentence capitalization here. 


`Create a Tenant from an existing Tenant`